hatch.toml

```
@@ -47,6 +47,22 @@ test = "pytest --no-cov {args:test/integ} -vvv --numprocesses=1"
 test_submitters = "pytest --no-cov {args:test/integ} -vvv --numprocesses=1 -m submitter"
 test_adaptors = "pytest --no-cov {args:test/integ} -vvv --numprocesses=1 -m adaptor"
 
+[envs.integ-ci]
+pre-install-commands = [
+  "pip install -r requirements-integ-testing.txt",
+  "pip install -r requirements-testing.txt"
```

We should to combine these into a single `pip install` so that the `pip` solver can satisfy both closures in a single pass. The [docs](https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-r) say:

>   This option can be used multiple times.

So, we would have:

```suggestion
  "pip install -r requirements-integ-testing.txt -r requirements-testing.txt"
```

---

Now one line.